ingest-app/src/migrations/009_update_bulk_status_message.clj

```diff
@@ -0,0 +1,20 @@
+(ns migrations.009-update-bulk-status-message
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 9."
+  []
+  (println "migrations.009-update-bulk-status-message up...")
+  (j/db-do-commands (config/db)
+                    "ALTER TABLE bulk_update_coll_status MODIFY
+                     (status_message VARCHAR2(4000))"))
```

Comment:
I forgot the details why I did it the way I did in 006_update_bulk_update_tables.clj. I just remember that whenever you change the column size, it's a non-trivial thing.  So I'm not sure a simple alter table will do the trick.

Reply:
Increasing the size of a column with the same type should be fine. Looking at the other migration you were changing the task_id column from a number to a varchar, which cannot be done in place.

When I read that migration I see the all of the bulk update tables are using VARCHAR rather than VARCHAR2. I'm not sure if you'll have an issue changing to VARCHAR2 so you can just leave it as VARCHAR.